db/migrate/20171026082635_add_task_action.rb

```diff
@@ -0,0 +1,9 @@
+class AddTaskAction < ActiveRecord::Migration
```

Comment:
Could you please generate the migration using our plugin specific task? See our [wiki](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Add-new-migration) for reference.

TL;DR: Please rename the migration to `20171026082635_add_task_action.foreman-tasks.rb`. It will be the first step to dynamic add and remove plugins.